src/tests/conformance/handshake.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
 async fn handshake_when_node_receives_connection() {
     // ZG-CONFORMANCE-001
 
-    // crate::tools::synthetic_node::enable_tracing();
+    // crate::tools::inner_node::enable_tracing();
```

Comment:
We should probably keep this function implemented on the `synthetic_node` and not the `inner_node`. The latter should ideally not really be exposed for simplicity.

Reply:
Okay, I'll move it then to synth_node.rs.
I checked, and it was added in inner_node.rs a long time ago. Maybe back then synth_node.rs was not yet ready.

Reply:
It was renamed and restructured a while back, we missed it then.

Reply:
Done